src/olympia/hero/models.py

```diff
@@ -113,6 +113,14 @@ def preview_image(self):
     preview_image.short_description = "Image"
     preview_image.allow_tags = True
 
+    def delete(self, *args, **kwargs):
+        (path, fn) = os.path.split(self.custom_image.path)
+        dest_thumb = path + b'/thumbs/' + fn
```

Comment:
I think as we're doing this in two places now (both here and in `PrimaryHeroImage.save_model`) we could do with abstracting it out into an `@property` (`def thumbnail_path`?) and then using that here and in `save_model`.